justfile

```diff
@@ -18,14 +18,31 @@ run-script script_name flags='' sig='' args='':
     -vvvv {{flags}}
   mv _test test
 
-run-deploy-voting-module-script flags: (run-script 'DeployLlamaTokenVotingModule' flags '--sig "run(address,string)"' '$SCRIPT_DEPLOYER_ADDRESS "tokenVotingModuleConfig.json"')
+# Token voting module
 
 dry-run-deploy: (run-script 'DeployLlamaTokenVotingFactory')
 
 deploy: (run-script 'DeployLlamaTokenVotingFactory' '--broadcast --verify --slow --build-info --build-info-path build_info')
 
+verify: (run-script 'DeployLlamaTokenVotingFactory' '--verify --resume')
+
+run-deploy-voting-module-script flags: (run-script 'DeployLlamaTokenVotingModule' flags '--sig "run(address,string)"' '$SCRIPT_DEPLOYER_ADDRESS "tokenVotingModuleConfig.json"')
+
 dry-run-deploy-voting-module: (run-deploy-voting-module-script '')
 
 deploy-voting-module: (run-deploy-voting-module-script '--broadcast --verify')
 
-verify: (run-script 'DeployLlamaTokenVotingFactory' '--verify --resume')
+# Account multicall module
+
+dry-run-deploy-acount-multicall-factory: (run-script 'DeployLlamaAccountMulticallFactory')
```

Comment:
`acount` should be `account` for all of these

Reply:
Done in [681bf1f](https://github.com/llamaxyz/llama-periphery/pull/100/commits/681bf1f43330c1098e1d2c25e90fbc907f2d140e)